ledger-service/http-json/src/test/scala/com/digitalasset/http/WebsocketServiceIntegrationTest.scala

```diff
@@ -526,6 +522,86 @@ class WebsocketServiceIntegrationTest
         }: Future[Assertion]
   }
 
+  "query on a bunch of random splits should yield consistent results" in withHttpService {
+    (uri, _, _) =>
+      val splitSample = SplitSeq.gen.map(_ map (BigDecimal(_))).sample.get
```

Comment:
As @leo-da has noted in the past, `GeneratorDrivenPropertyChecks` and `Async*Spec` do not play well together. I would like to do something better than "just take one sample and test it", though.